plugins/One Click Lyrics/src/index.ts

```diff
@@ -15,15 +15,32 @@ const icon =
 </svg>`;
 
 let addedElement: HTMLButtonElement | null = null;
-let observer: MutationObserver | null = null;
 
 const lyricsLabel = "Lyrics";
 const lyricsNotAvailableLabel = "No lyrics available";
+const buttonId = "oneClickLyricsButton";
+const volumeContainerId = "._sliderContainer_15490c0";
+
+observePromise(unloads, volumeContainerId).then((volumeContainer) => {
+    if (!volumeContainer || addedElement) return;
+
+    addedElement = createButton();
+    (volumeContainer as HTMLDivElement).before(addedElement);
+
+    // On plugin load, check current song and update button state using lyrics from redux store
+    const state = redux.store.getState();
+    const currentTrack = state.playQueue.elements[state.playQueue.currentIndex];
+    const currentTrackId = currentTrack?.mediaItemId;
+    
+    MediaItem.fromId(currentTrackId).then(async (mediaItem) => {
```

Comment:
You should use `MediaItem.fromPlaybackContext` here.

https://github.com/Inrixia/TidaLuna/blob/562304a9dc99a66926ee01518441e473ca3cddba/plugins/lib/src/classes/MediaItem/MediaItem.ts#L75